packages/bitcore-lib/test/publickey.js

```diff
@@ -451,4 +451,36 @@ describe('PublicKey', function() {
 
   });
 
+  describe('#isValidTaproot', function() {
```

Comment:
These are just some ideas. Consider replacing string values in 'isValidTaproot' inputs below with named constants defined in the describe block.
`
    const VALID_TAPROOT = '29b21ed3959615c97d0866a295486c25577aaa5ee18d463489d9dbe3cf0aaf5e';
    
    const INVALID_TAPROOT = 'bca10d4006dbfebe31eee345df1d18d738f72fcd62d6ccafa84f87f05bac3467'; // Why is this an invalid hex string?
    
    const INVALID_LENGTH = VALID_TAPROOT.slice(0, -1); // String length is different from valid
    
    const INVALID_HEX_DIGIT = 'g'; // Documents example of why a "hex string" may be invalid
    
    const INVALID_HEX = INVALID_LENGTH + INVALID_HEX_DIGIT;
    
`